argo-cd-apps/base/monitoring-workload-in-cluster.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: monitoring-workload-in-cluster
+
+spec:
+  project: workload-monitoring
```

Comment:
project must be `default`

Reply:
Currently only default project is used. I am ok with creation of new projects, but that must be created probably in bootstrap script, not sure if can be created by ArgoCD using `default` project.

Reply:
done